src/Compilers/VisualBasic/Test/Semantic/HasValidFonts.vb

```diff
@@ -12,7 +12,12 @@ Friend Class HasValidFonts
     Public Overrides ReadOnly Property ShouldSkip As Boolean
         Get
             Try
+#If NET8_0 Then
+                If OperatingSystem.IsWindowsVersionAtLeast(6, 1) Then
```

Comment:
This check is needed in .NET TFM, because `System.DefaultFont` is guarded by CA1416
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1416
to be only available on Windows 6.1 or later